src/bcf/report/oncoprint.rs

```diff
@@ -368,7 +369,10 @@ pub fn oncoprint(
                         rec.extend(record_tuples.iter().map(|(r, _)| r.clone()));
                     }
                     1 => rec.extend(filter_canonical.iter().map(|(r, _)| r.clone())),
-                    _ => panic!("Found more than one variant annotated as canonical!"),
+                    _ => {
+                        rec.extend(filter_canonical.iter().map(|(r, _)| r.clone()));
+                        warn!("Found more than one variant annotated as canonical!");
```

Comment:
Please add some info identifying the record here: CHROM, POS, and gene name of the problematic entries.

Reply:
I've added the gene name, positions and the alterations that are marked as canonical. `CHROM` would definitely need a bit more work in order to be included in the warning.